src/py/flwr/common/message.py

```diff
@@ -63,5 +149,47 @@ class Message:
         logic to a client, or vice-versa) or that will be sent to it.
     """
 
-    metadata: Metadata
+    _metadata: Metadata
     content: RecordSet
+
+    def __init__(self, metadata: Metadata, content: RecordSet) -> None:
+        self._metadata = metadata
+        self.content = content
+
+    @property
+    def metadata(self) -> Metadata:
+        """A dataclass including information about the message to be executed."""
+        return self._metadata
+
+    def create_reply(self, content: RecordSet, ttl: str) -> "Message":
```

Comment:
why `-> "Message":` instead of `-> Message:`?

Reply:
I tried to use Message without quotes, but got error saying Message is not defined.

Reply:
You can add at the top this line and then it will work w/o the `""`:
```
from __future__ import annotations
```

Reply:
Cool, thanks